.github/workflows/python-tests.yml

```diff
@@ -226,6 +226,8 @@ jobs:
           python-version: "3.11"
       - name: Setup virtual env
         uses: ./.github/actions/make_init
+        with:
```

Comment:
it might be a good idea to add a brief comment here why we don't want to cache

Reply:
Added 👍